setup.py

```diff
@@ -16,5 +16,5 @@
           'pip==18.0',
           'requests==2.14.2'
       ],
-      packages=['mibitracker']
+      packages=['mibitracker', 'mibitof']
```

Comment:
Does this mean that I can import the package mibitof from here ? If so, what if I want to import both this mibitof and the other mibitof in the mibitof repository?

Reply:
Yes, that is the point. I'm not sure if there will be a naming conflict. I guess we could come up with a different name for this module (mibiprocessing? mibidata?) and then keep mibitof intact.

Reply:
I guess that would be better. Another question I have (just curious): when we add ```git+git://github.com/ionpath/mibitracker-client@master``` to environment.yml, does it automatically install both of these modules? I'm just trying to understand how this works.

Reply:
Yes, when pip clones it from git, it runs setup.py which builds the two modules (as well as any others we add in the future). And then it's added it to the path (or some sort of other magic happens) and can just be called as `from mibitof import mibi_image as mi`, etc.

Reply:
OK, I see that the other mibitof has ```packages=['mibitof']``` statement, so there is a high chance that there will be a naming conflict (maybe you should try it out). Maybe naming as you suggested could be useful and safer.

Reply:
Turns out, yes, there was a naming conflict and so I switched this from `mibitof` to `mibidata`.